settings/__init__.py

```diff
@@ -30,3 +30,4 @@
 if len(sys.argv) > 1 and sys.argv[1] == 'test':
     # stuff that's absolutely required for a test run
     CELERY_ALWAYS_EAGER = True
+    INSTALLED_APPS += ('django_nose',)
```

Comment:
I think so. One of the neat things about `pytest-django` is that it required no changes to the app. They can coexist. I don't think that's necessary, but it's still kinda cool. I think I will (should) remove the nose stuff anyway though.